torchbox/torchbox.com
# delete templates which are no longer referenced by code

[Link to Ticket](https://torchbox.monday.com/boards/1192293412/pulses/1361577420)

### Description of Changes Made

As part of the clean-up before the data import, we removed some page models but didn't remove the associated templates: culture page, values page, job index page and impact report.

### How to Test
Check the list of removed templates and ensure they are not referenced anywhere in the codebase.

### Screenshots

<details>
  <summary>Expand to see more</summary>

</details>

### MR Checklist


#### Unit tests


#### Documentation


#### Browser testing

  - Latest version of Chrome on mac
  - Latest version of Firefox on mac
  - Latest version of Safari on mac
  - Safari on last two versions of iOS
  - Chrome on last two versions of Android

#### Data protection


#### Accessibility


#### Sustainability
